WordPress/Classes/Models/Blog.h

```diff
@@ -226,6 +226,7 @@ typedef NS_ENUM(NSInteger, SiteVisibility) {
 - (BOOL)supports:(BlogFeature)feature;
 - (BOOL)supportsPublicize;
 - (BOOL)supportsShareButtons;
+- (BOOL)supportsStats;
```

Comment:
Wdyt about renaming to `isStatsActive` or something like that?

I believe most sites "support" it but might not have it "active", which is a minor subtle difference.

Reply:
Fixed! 85b43e6